aggregator/src/test/java/org/jenkinsci/plugins/workflow/WorkflowRunTest.java

```diff
@@ -201,4 +211,48 @@ public void contextInjectionOfSubParameters() throws Exception {
         r.assertBuildStatusSuccess(JenkinsRuleExt.waitForCompletion(b));
     }
 
+    @Issue("JENKINS-25550")
+    @Test public void hardKill() throws Exception {
+        p = r.jenkins.createProject(WorkflowJob.class, "p");
+        p.setDefinition(new CpsFlowDefinition("zombie()"));
+        WorkflowRun b = p.scheduleBuild2(0).waitForStart();
+        JenkinsRuleExt.waitForMessage("undead", b);
+        Executor ex = b.getExecutor();
+        assertNotNull(ex);
+        ex.interrupt();
+        JenkinsRuleExt.waitForMessage("bwahaha org.jenkinsci.plugins.workflow.steps.FlowInterruptedException #1", b);
+        ex.interrupt();
+        JenkinsRuleExt.waitForMessage("bwahaha org.jenkinsci.plugins.workflow.steps.FlowInterruptedException #2", b);
+        ex.interrupt();
+        JenkinsRuleExt.waitForMessage("Hard kill!", b);
+        JenkinsRuleExt.waitForCompletion(b);
+        r.assertBuildStatus(Result.ABORTED, b);
+    }
+    public static class Zombie extends AbstractStepImpl {
```

Comment:
add new line. With `@Test` without new line this block of code looks very unparsable by eyes.